src/main/java/org/opentestsystem/shared/contentspecid/ContentSpecIdBuilder.java

```diff
@@ -108,7 +108,7 @@ public ContentSpecId build() throws IdParseException {
         validateEmphasis(this.emphasis, emphasis);
 
         String contentCategory = deriveContentCategory(this.subject, convertedClaim, emphasis, convertedDomain);
-        validateContentCategory(this.contentCategory, contentCategory);
+        validateContentCategory(this.target, this.contentCategory, contentCategory);
```

Comment:
So target is now required for content category when it wasn't prior to this change?

Reply:
Yes, the way the test package files have the IDs built up from successive parts (see sample below), at certain stages they have a content category appropriate to the target (P in the example), but haven't specified a target yet. That causes a mismatch between what is in the string and what is derived for content category. That situation doesn't occur in any of the library test files. 

<BlueprintElement id="1" type="claim" label="M.G7.C1">
      <BlueprintElement id="1|P" type="target" label="M.G7.C1">
        <BlueprintElement id="1|P|TS01" type="target" label="M.G7.C1">
          <BlueprintElement id="1|P|TS01|A-7" type="target" label="M.G7.C1RP.TA"/>
        </BlueprintElement>
      </BlueprintElement>
    </BlueprintElement>